### DISCUSSION TITLE
Alert when Revisor edits a post they've already revised

### DISCUSSION THREAD

**Discussion by agapetry** [publishpress/PublishPress-Revisions#915] [Feature Requests]
When a Revisor edits a post that they've already submitted a revision for, display a notice with a link to that revision.

**Comment by KratosGemini** [comment]
I think this might be done now.

![image](https://github.com/publishpress/PublishPress-Revisions/assets/1143805/c426450e-0315-404d-a818-e0dbd9cf640b)
